Build fails on filesystems where xattr is not supported

When trying to build buildkit on a filesystem where xattr support is unavailable or disallowed, the following error occurs:

```
$ make
mkdir -p bin
docker build -t buildkit:buildkitd --target buildkitd -f ./hack/dockerfiles/test.Dockerfile --force-rm .
[+] Building 0.1s (2/2) FINISHED                                                
 => ERROR local://context (.dockerignore)                                  0.0s
 => => transferring context: 89B                                           0.0s
 => CANCELED local://dockerfile (test.Dockerfile)                          0.1s
 => => transferring dockerfile:                                            0.0s
------
 > local://context (.dockerignore):
------
error from sender: failed to xattr .dockerignore: failed to xattr .dockerignore: operation not supported
make: *** [bin/buildkitd] Error 1
```

Additionally, this issue affects running buildkit on filesystems where xattr is not supported and (for some reason) prevents the use of the experimental docker buildkit feature on CoreOS.

--- a/vendor.conf
+++ b/vendor.conf
@@ -39,7 +39,7 @@ golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
 github.com/docker/docker 71cd53e4a197b303c6ba086bd584ffd67a884281
 github.com/pkg/profile 5b67d428864e92711fcbd2f8629456121a56d91f
 
-github.com/tonistiigi/fsutil b19464cd1b6a00773b4f2eb7acf9c30426f9df42
+github.com/tonistiigi/fsutil 7e391b0e788f9b925f22bd3cf88e0210d1643673
 github.com/hashicorp/go-immutable-radix 826af9ccf0feeee615d546d69b11f8e98da8c8f1 git://github.com/tonistiigi/go-immutable-radix.git
 github.com/hashicorp/golang-lru a0d98a5f288019575c6d1f4bb1573fef2d1fcdc4
 github.com/mitchellh/hashstructure 2bca23e0e452137f789efbc8610126fd8b94f73b

